web/views/shared_view.ex

```diff
@@ -144,10 +144,10 @@ defmodule CoursePlanner.SharedView do
   end
 
   def form_datetime(form, field, opts \\ []) do
-    value =
+    value = Map.get(form.source.changes, field) ||
```

Comment:
I simplified it but basically when you wanna merge two map it won't accept nil as input, that's why i wrote the case,